### PR TITLE
add 02-where

### PR DESCRIPTION
**Pull request by knaufk**

### CODE REVIEW

File: recipes/02/02_where.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Script
+
+```sql
+-- This example will show how to filter server logs 
+-- in real-time using a standard WHERE clause.
+-- The table it uses is backed by the `faker` connector 
+-- (https://github.com/knaufk/flink-faker) which 
+-- generates rows in memory and is convenient for testing
+-- queries. As such, it is an alternative to the built-in 
+-- `datagen` connector for testing or prototyping queries.
```
Comment (morsapaes) [inline recipes/02/02_where.md:12]: Could we move the explanations to the top-level, instead of having them as comments? I find this distracting.

Comment (knaufk) [reply recipes/02/02_where.md:12]: I assume, we would you also do this for the CREATE TABLE example then?

File: recipes/02/02_where.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  'connector' = 'faker', 
+  'fields.client_ip.expression' = '#{Internet.publicIpV4Address}',
+  'fields.client_identity.expression' =  '-',
+  'fields.userid.expression' =  '-',
+  'fields.log_time.expression' =  '#{date.past ''15'',''5'',''SECONDS''}',
+  'fields.request_line.expression' = '#{regexify ''(GET|POST|PUT|PATCH){1}''} #{regexify ''(/search\.html|/login\.html|/prod\.html|cart\.html|/order\.html){1}''} #{regexify ''(HTTP/1\.1|HTTP/2|/HTTP/1\.0){1}''}',
+  'fields.status_code.expression' = '#{regexify ''(200|201|204|400|401|403|301){1}''}',
+  'fields.size.expression' = '#{number.numberBetween ''100'',''10000000''}'
+);
+-- You can filter this (unbounded) table of
```
Comment (morsapaes) [inline recipes/02/02_where.md:31]: Just a suggestion, but it'd sound simpler to add "which **continuously** generates rows in memory ..." to the previous explanation and get rid of the "(unbounded)" here.

**Comment by morsapaes** [comment]
👍

File: recipes/02/02_where.md (hunk 1 above)
Comment (morsapaes) [reply recipes/02/02_where.md:12]: Ideally, yes. Makes it look a bit cleaner and easier to follow, in my opinion.

**Comment by sjwiesman** [comment]
Agree with Marta's comments otherwise +1

File: recipes/02/02_where.md (hunk 1 above)
Comment (knaufk) [reply recipes/02/02_where.md]: Thanks. Done :)

File: recipes/02/02_where.md (hunk 2 above)
Comment (knaufk) [reply recipes/02/02_where.md]: I rewrote this.

**Pull request merged**